app_scraping/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from django.utils import timezone
+
+
+# Create your models here.
+class SkiResort(models.Model):
+    resort_name = models.CharField(max_length=200, unique=True)
+    trails_open = models.IntegerField(default=0)
+    lifts_open = models.IntegerField(default=0)
+    acres_open = models.IntegerField(default=0)
+    terrain_percent = models.IntegerField(default=0)
+
+    # string of total trails, ex. "/ 128"
+    total_trails = models.CharField(max_length=200, default='0')
+    # string of total lifts, ex. "/ 21"
+    total_lifts = models.CharField(max_length=200, default='0')
+
+    '''
```

Comment:
Avoid merging dead (commented-out) code. If you want to keep track of when the model was created/updated, inherit from `TimeStampedModel` instead of `models.Model`. It's a part of a third-party package: https://django-extensions.readthedocs.io/en/latest/model_extensions.html.

Reply:
Removed dead code.